packages/twenty-server/.env.example

```diff
@@ -73,3 +73,7 @@ SIGN_IN_PREFILLED=true
 # MUTATION_MAXIMUM_RECORD_AFFECTED=100
 # CHROME_EXTENSION_ID=bggmipldbceihilonnbpgoeclgbkblkp
 # PG_SSL_ALLOW_SELF_SIGNED=true
+# OPENAI_API_KEY=sk-proj-abcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcd
+# LANGFUSE_SECRET_KEY=sk-lf-abcdabcd-abcd-abcd-abcd-abcdabcdabcd
```

Comment:
These one you can document in self-hosting-var.mdx but I don't think they should be in .env.example - we only keep the most important ones for self-hosting here